src/OpenTelemetry.Api/Trace/ActivityExtensions.cs

```diff
@@ -0,0 +1,57 @@
+﻿// <copyright file="ActivityExtensions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics;
+using System.Linq;
+
+namespace OpenTelemetry.Trace
+{
+    /// <summary>
+    /// Extension methods on Activity.
+    /// </summary>
+    public static class ActivityExtensions
+    {
+        private const string StatusCodeKey = "ot.status_code";
```

Comment:
You should be able to use `SpanAttributeConstants.StatusCodeKey` & `SpanAttributeConstants.StatusDescriptionKey`.